Sources/Frontend/Commands/ScanCommand.swift

```diff
@@ -57,6 +57,9 @@ struct ScanCommand: FrontendCommand {
     @Flag(help: "Retain all public declarations, recommended for framework/library projects")
     var retainPublic: Bool = defaultConfiguration.$retainPublic.defaultValue
 
+    @Option(parsing: .upToNextOption, help: "SPIs to check for unused code even when retain-public is enabled")
```

Comment:
```suggestion
    @Option(parsing: .upToNextOption, help: "SPIs (System Programming Interfaces) to check for unused code even when retain-public is enabled")
```

Many users likely won't know what SPI means.